streampipes-client-python/overrides/text.docs

```diff
@@ -0,0 +1,40 @@
+├─ assets/
```

Comment:
From my point of view the whole `text.docs` file is not required
Or what is the reason why you include it?